build.gradle

```diff
@@ -332,6 +332,8 @@ subprojects { subproj ->
         // Effectively disable JUnit concurrency by running tests in only one thread by default.
         systemProperty 'junit.jupiter.execution.parallel.config.strategy', 'fixed'
         systemProperty 'junit.jupiter.execution.parallel.config.fixed.parallelism', 1
+        // Order test classes by annotation.
+        systemProperty 'junit.jupiter.testclass.order.default', 'org.junit.jupiter.api.ClassOrderer$OrderAnnotation'
```

Comment:
This allows decorating test classes with `@Order(...)` annotations to help long-running classes get scheduled early on.